est/001.est

```diff
@@ -0,0 +1,22 @@
+date: 28-02-2016
+author: Dali Freire
+method: champions.pert
+scope:
+  1: Infrastructure setup
+  2: Design the operation commands
+  3: GitHub/Issue trackers integration
+  4: Project site
+  5: Integrate with continuous integration systems
+  6: Integration testing
+  7: Performance testing
+  8: Achieve 80% test coverage
+  9: User documentation
```

Comment:
@dalifreire This should not be part of the estimation as explained [here](https://github.com/teamed/est/blob/master/README.md).
>Coding Time Only. Estimate code writing time only. Don't estimate time you would spend on discussions, thinking, modeling, diagramming, documenting etc.